.github/workflows/rust.yml

```diff
@@ -46,8 +46,12 @@ jobs:
       run: echo RUSTFLAGS='-Dwarnings' >> $GITHUB_ENV
     - run: rustup update
     - run: cargo version
-    - run: cargo build --target ${{ matrix.target }}
-    - run: cargo test --target ${{ matrix.target }}
+    - uses: stellar/binaries@v30
+      with:
+        name: cargo-hack
+        version: 0.5.28
+    - run: cargo-hack hack build --feature-powerset --target ${{ matrix.target }}
+    - run: cargo-hack hack test --feature-powerset --target ${{ matrix.target }}
```

Comment:
I think we can revert this part of the change now that no additional features are added.

Reply:
I think we can keep the current way, the previous testing method will not test whether cli can be compiled normally.

Reply:
Ah good call. 👍🏻 💯 ❤️